apps/web-giddh/src/app/shared/directives/resizable.directive.ts

```
@@ -0,0 +1,430 @@
+      document.body.style.cursor = '';
+      document.body.style.userSelect = '';
+      document.body.style.pointerEvents = ''; // Restore pointer events
+      
+      // Release pointer capture if it was set
+      if (this.resizerElement && this.resizerElement.releasePointerCapture) {
+        try {
+          // We need to release all pointer captures, but we don't have the pointerId
+          // This is handled automatically when the mouse is released in most browsers
+        } catch (e) {
+          console.error('ResizableDirective: Failed to release pointer capture:', e);
+        }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)